### PR TITLE
Add a Dummy model for integration tests

### PR DESCRIPTION
**Pull request by jenny011**
This PR adds a Dummy model that has only an Identity layer to the models.
The Dummy model is used instead of ResNet18 in integration tests to reduce CI resource consumption.

### CODE REVIEW

File: modyn/models/dummy/dummy.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from modyn.models.coreset_methods_support import CoresetSupportingModule
+from torch import nn
+
+# from torchvision.models.resnet import BasicBlock, ResNet
```
Comment (MaxiBoether) [inline modyn/models/dummy/dummy.py:6]: we can remove this i guess

**Review by MaxiBoether — APPROVED** [review]
Thanks, just one minor question. After that is resolved, we can merge

File: integrationtests/supervisor/integrationtest_supervisor.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def test_start_two_experiment_pipelines() -> None:
     try:
         dataset_helper.setup_dataset()
         test_start_one_experiment_pipeline()
-        test_start_two_experiment_pipelines()
+        # test_start_two_experiment_pipelines()
```
[the remark this answers is not in the document]
Comment (jenny011) [reply integrationtests/supervisor/integrationtest_supervisor.py:87]: Ah, no. I should uncomment it.

File: modyn/models/dummy/dummy.py (hunk 1 above)
Comment (jenny011) [reply modyn/models/dummy/dummy.py]: done

File: integrationtests/supervisor/integrationtest_supervisor.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jenny011) [reply integrationtests/supervisor/integrationtest_supervisor.py]: done

**Pull request merged**